Webhook/WAWebhookNew.js

```diff
@@ -166,12 +169,18 @@ const handleTextMessage = async (message, from, phone_number_id) => {
         case types.employee.Offer:
             if ((session && session.targetAgent)) {
                 if (session.agentType == 'assistant') {
-                    if ((session.payment && session.payment.transaction.status == 'success') || phone_number_id == '359476970593209')//payment found in redis session
-                    {
+                    if(session.interactions <= 10 || session.payment.transaction.status == 'success' || phone_number_id == '359476970593209'){
                         response = await interactWithAssistant(message.text.body, from, session.targetAgent.assistantId, session.threadId);
                         if (response.answer && response.answer != '')
                             response.answer = convertMarkdownToWhatsApp(response.answer);
                     }
+                    else if ((session.interactions > 10 && session.payment && session.payment.transaction.status == 'pending') || phone_number_id == '359476970593209')//payment found in redis session
```

Comment:
here, if instead of paying customer sends any other message, then this condition is still valid and it will send new payment order.
add a check for it